rust/pact_matching/src/tests.rs

```diff
@@ -183,7 +183,21 @@ fn matching_headers_be_false_when_headers_are_not_equal() {
     assert!(!mismatches.is_empty());
     assert_eq!(mismatches[0], Mismatch::HeaderMismatch { key: "HEADER".to_string(),
         expected: "HEADER".to_string(), actual: "HEADER2".to_string(),
-        mismatch: "Expected header 'HEADER' to have value 'HEADER' but was 'HEADER2'".to_string() });
+        mismatch: "".to_string() });
```

Comment:
I created a simple macro (you'll find it in lib.rs) so you don't need to do `.to_string()` on string slices. You can do `s!("")` instead.